src/stores/logs.ts

```diff
@@ -18,14 +18,21 @@ export const useLogStore = defineStore(
     const lastFetchedIp = ref<string>()
     const geoDetails = ref<string | null>(null)
     const ispDetails = ref<string | null>(null)
-
+    const reverseSort = ref<boolean>(false)
     const logTypeFilter = ref<LogType[]>([LogType.NORMAL, LogType.INFO, LogType.WARNING, LogType.CRITICAL])
     const logMessageFilter = ref('')
 
     const filteredLogsByType = computed(() => logs.value.filter(log => logTypeFilter.value.includes(log.type)))
     const { results: filteredLogs } = useSearchQuery(filteredLogsByType, logMessageFilter, log => log.message)
-    const { paginatedResults, currentPage, pageCount } = useArrayPagination(filteredLogs, 30)
+    
+    const finalLogs = computed(() => {
+      return reverseSort.value ? [...filteredLogs.value].reverse() : [...filteredLogs.value];
+    });
```

Comment:
Prevent duplicating array if not needed

```suggestion
    const finalLogs = computed(() => reverseSort.value ? filteredLogs.value.toReversed() : filteredLogs.value)
```

Reply:
removed duplicate array.

as of using useSorted i am getting into type  mis-match issue  with simple implementation. (dont want to complicate code).

so went with sorting in place. (Hope this will be file )

Reply:
VueUse composables returns new refs so you have to create a new constant.
You're good with this, I'll push what's needed.